app/uk/gov/hmrc/softdrinksindustrylevystub/models/package.scala

```diff
@@ -77,4 +78,11 @@ package object models {
     }
   }
 
+  private[models] implicit class ValidationOptionLitresProduced(lp: Option[LitresProduced]) {
+    def isValid: Boolean = lp match {
+      case Some(a) => a.isValid
+      case _ => true
```

Comment:
consider - 
```
lp.map(_.isValid).getOrElse(true)
```

Reply:
Changed to..

def isValid: Boolean = lp.forall(_.isValid)